piv/pcsc_unix.go

```diff
@@ -18,14 +18,11 @@ package piv
 
 // https://ludovicrousseau.blogspot.com/2010/04/pcsc-sample-in-c.html
 
-// TODO: Figure out if linux flags should use pkg-config instead
-
 // #cgo darwin LDFLAGS: -framework PCSC
-// #cgo linux CFLAGS: -I/usr/include/PCSC
-// #cgo linux LDFLAGS: -lpcsclite
-// #cgo freebsd CFLAGS: -I/usr/local/include/
-// #cgo freebsd CFLAGS: -I/usr/local/include/PCSC
-// #cgo freebsd LDFLAGS: -L/usr/local/lib/
+// #cgo linux pkg-config: libpcsclite
+// #cgo freebsd CFLAGS: -I/usr/local/include/	
```

Comment:
There's trailing whitespace on these lines, can you remove that so there's no diff for the freebsd cgo lines?